csharp/Platform.Ranges/Range[T].cs

```diff
@@ -58,89 +70,144 @@ public Range(T minimum, T maximum)
+        /// <para>Presents the <see cref="Range{T}"/> in readable format.</para>
+        /// <para>Представляет <see cref="Range{T}"/> в удобном для чтения формате.</para>
         /// </summary>
-        /// <returns><para>String representation of the Range.</para><para>Строковое представление диапазона.</para></returns>
+        /// <returns>
+        /// <para>String representation of the <see cref="Range{T}"/>.</para>
+        /// <para>Строковое представление <see cref="Range{T}"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public override string ToString() => $"[{Minimum}..{Maximum}]";
 
         /// <summary>
-        /// <para>Determines if the provided value is inside the range.</para>
-        /// <para>Определяет, находится ли указанное значение внутри диапазона.</para>
+        /// <para>Determines if the <see cref="Range{T}"/> contains the <paramref name="value"/>.</para>
+        /// <para>Определяет, содержит ли <see cref="Range{T}"/> <paramref name="value"/>.</para>
         /// </summary>
-        /// <param name="value"><para>The value to test.</para><para>Значение для проверки.</para></param>
-        /// <returns><para>True if the value is inside Range, else false.</para><para>True, если значение находится внутри диапазона, иначе false.</para></returns>
+        /// <param name="value">
+        /// <para>A value to find in the <see cref="Range{T}"/>.</para>
+        /// <para>Значение, которое нужно найти в <see cref="Range{T}"/>.</para>
+        /// </param>
+        /// <returns>
+        /// <para>A <see cref="Boolean"/> value that determines whether the <see cref="Range{T}"/> contains the <paramref name="value"/>.</para>
+        /// <para>Значение типа <see cref="Boolean"/>, определяющее содержит ли <see cref="Range{T}"/> <paramref name="value"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool Contains(T value) => _comparer.Compare(Minimum, value) <= 0 && _comparer.Compare(Maximum, value) >= 0;
 
         /// <summary>
-        /// <para>Determines if another range is inside the bounds of this range.</para>
-        /// <para>Определяет, находится ли другой диапазон внутри границ этого диапазона.</para>
+        /// <para>Determines if the current <see cref="Range{T}"/> contains the <paramref name="range"/>.</para>
+        /// <para>Определяет, содержит ли текущий <see cref="Range{T}"/> <paramref name="range"/>.</para>
         /// </summary>
-        /// <param name="range"><para>The child range to test.</para><para>Дочерний диапазон для проверки.</para></param>
-        /// <returns><para>True if range is inside, else false.</para><para>True, если диапазон находится внутри, иначе false.</para></returns>
+        /// <param name="range">
+        /// <para>A <see cref="Range{T}"/> to find in the current <see cref="Range{T}"/>.</para>
```

Comment:
`A <see cref="Range{T}"/> to test for inclusion in the current <see cref="Range{T}"/>.`